X10D/src/Drawing/ColorExtensions.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Diagnostics.Contracts;
 using System.Drawing;
 using System.Runtime.CompilerServices;
+using X10D.Collections;
```

Comment:
This seems to be the only change made in this file. Was there a reason this namespace is imported?